flink-table/flink-table-api-java/src/test/java/org/apache/flink/table/catalog/CatalogTestBase.java

```diff
@@ -168,7 +168,7 @@ protected Map<String, String> getBatchTableProperties() {
         return new HashMap<String, String>() {
             {
                 put(IS_STREAMING, "false");
-                putAll(getGenericFlag(isGeneric()));
+                put(FactoryUtil.CONNECTOR.key(), "hive");
```

Comment:
as this is now only for hive - does this generic class need to about this option - can't this be done in hive specific code or in a way that doesn ot hard code a "hive" literal in a non-hive class.